<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>factor</class>
 <widget class="QMainWindow" name="factor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1060</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 20pt &quot;Agency FB&quot;;
border-image: url(:/20181210_211055.jpg);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>401</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>16</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Agency FB&quot;;
color: rgb(0, 0, 0);
border-image: url(:/20181210_210744.jpg);</string>
    </property>
    <property name="text">
     <string>Here Is Your Factor</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="clientnamefinal">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>100</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 112);
border-image: url();
background-color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);
border-color: rgb(0, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Agency FB&quot;;
color: rgb(0, 0, 0);
border-image: url(:/20181210_210744.jpg);</string>
    </property>
    <property name="text">
     <string>Client Name :</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listofbooksfinal">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>170</y>
      <width>381</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url();
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 127);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Agency FB&quot;;
color: rgb(0, 0, 0);
border-image: url(:/20181210_210744.jpg);</string>
    </property>
    <property name="text">
     <string>your books</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>330</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Agency FB&quot;;
color: rgb(0, 0, 0);
border-image: url(:/20181210_210744.jpg);</string>
    </property>
    <property name="text">
     <string>total cost</string>
    </property>
   </widget>
   <widget class="QLabel" name="totalc">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>320</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 255, 255);
border-image: url();
color: rgb(0, 0, 127);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="back_factor">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>-10</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Agency FB&quot;;
color: rgb(0, 0, 0);
border-image: url(:/20181210_210744.jpg);</string>
    </property>
    <property name="text">
     <string>back</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>430</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Agency FB&quot;;
color: rgb(0, 0, 0);
border-image: url(:/20181210_210744.jpg);</string>
    </property>
    <property name="text">
     <string>Thanks For Your Coming... :)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1060</width>
     <height>46</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
